modules/installation-requirements-user-infra.adoc

```diff
@@ -80,6 +102,65 @@ Each cluster machine must meet the following minimum requirements:
 
 |===
 
+ifeval::["{context}" == "installing-ibm-z"]
+[id="minimum-ibm-z-system-requirements_{context}"]
+== Minimum IBM Z system requirements
+
+=== Hardware requirements
+
+* 1 LPAR with 3 IFLs that supports SMT2
+* 1 OSA or RoCE network adapter, or both
+
+=== Operating system requirements
+
+* One instance of z/VM 7.1
+
+On your z/VM instance, set up:
+
+* 3 guest virtual machines for OCP control plane machines (master nodes)
+* 3 guest virtual mchines  for OCP compute machines (worker nodes)
+* 1 guest virtual machine for the temporary OCP bootstrap machine
+
+=== Disk storage for the z/VM guest virtual machines 
+
+* FICON attached disk storage (DASDs). These can be z/VM minidisks, fullpack minidisks, or dedicated DASDs. If available, use HyperPAV to ensure optimal performance.
+* FCP attached disk storage
+
+=== Storage / Main Memory
+
+* 16 GB for OCP control plane machines (master nodes)
+* 8 GB for OCP compute machines (worker nodes)
+* 16 GB for the temporary OCP bootstrap machine
+
+== Preferred IBM Z system requirements
+
+=== Hardware requirements
+
+* 3 LPARs with 6 IFLs that support SMT2
+* 1 or 2 OSA or RoCE network adapters, or both
+
+=== Operating system requirements
+
+* 2 or 3 instances of z/VM 7.1 for high availability
+
+On your z/VM instances, set up:
+
+* 3 guest virtual machines for OCP control plane machines, one per z/VM instance
+* At least 6 guest virtual mchines for OCP compute machines, distributed across the z/VM instances
+* 1 guest virtual machine for the temporary OCP bootstrap machine
+
+=== Disk storage for the z/VM guest virtual machines 
+
+* FICON attached disk storage (DASDs). These can be z/VM minidisks, fullpack minidisks, or dedicated DASDs. If available, use HyperPAV to ensure optimal performance.
+* FCP attached disk storage
+
+=== Storage / Main Memory
+
+* 16 GB for OCP control plane machines (master nodes)
+* 8 GB for OCP compute machines (worker nodes)
+* 16 GB for the temporary OCP bootstrap machine
+endif::[]
+
 [id="csr_management_{context}"]
 == Certificate signing requests management
```

Comment:
This section seems like it could use a link to how one is supposed approve the CSRs. I.e. See 7.1.12 for details.

Reply:
@kalexand-rh What is the correct place for xref in this case, please? Is it in the assembly or in the module?

Reply:
@zzoubkova, you can't put an xref in the module, and unless something's changed, the process in 7.1.12 is not the same as the the process described in this section.

Reply:
ok, thank you for info